src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -143,6 +183,78 @@ def post_process(self, outputs, target_sizes):
 
         return results
 
+    def post_process_one_shot_object_detection(self, outputs, threshold=0.6, nms_threshold=0.3, target_sizes=None):
+        """
+        Converts the output of [`OwlViTForObjectDetection.image_guided_detection`] into the format expected by the COCO
+        api.
+
+        Args:
+            outputs ([`OwlViTImageGuidedObjectDetectionOutput`]):
+                Raw outputs of the model.
+            threshold (`float`, *optional*, defaults to 0.6):
+                Minimum confidence threshold to use to filter out predicted boxes.
+            nms_threshold (`float`, *optional*, defaults to 0.3):
+                IoU threshold for non-maximum suppression of overlapping boxes.
+            target_sizes (`torch.Tensor`, *optional*, defaults None):
+                Tensor containing the size (h, w) of each image of the batch. For evaluation, this must be the original
+                image size (before any data augmentation). For visualization, this should be the image size after data
+                augmentation, but before padding.
```

Comment:
This docstring can probably be updated (not sure the sentence "after data augmentation, but before padding" is relevant here (this was the case for DETR)